src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', resolve);
+});
+
+promise1.then(onSuccess, onError);
```

Comment:
It's better to handle `onError` in `catch` block because you may have any unexpected errors not only rejected promises.